package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "5.5.1",
+  "version": "5.5.2",
```

Comment:
Also commit package-lock.json please.

Reply:
`pnpm-lock.yaml` ahh ok

Reply:
![image](https://github.com/bcgov/namerequest/assets/3484109/75c9a1b2-05c4-4dfa-ac3b-dd8cdeae608c)

Reply:
Oh yeah. Please and thanks!

Reply:
I'm leaving this conversation open until I see the pnpm-lock.yaml file in this PR. Thanks.

Reply:
I don't think it needs an update, let me check the other PRs?

Reply:
Yeah it doesn't need an update? Looking from your example here:

https://github.com/bcgov/namerequest/pull/764

Reply:
I see that you're right! I thought the app version was in there but I guess it's not. Resolving.